pkg/integrations/v2/vmware_exporter/vmware_exporter.go

```diff
@@ -0,0 +1,97 @@
+package vmware_exporter
```

Comment:
nit: consistency between package/file name and integration name?

Reply:
@marctc I named the package after the project but we could change this if necessary. My thinking was that eventually the exporter could container collectors for things besides vSphere, such as NSX.

Reply:
Gotcha, thanks